### PR TITLE
Add a description to the light system channel types

### PR DESCRIPTION
**Pull request by lolodomo**
Sets the category to DimmableLight for the brightness channel type.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties [hunk 1]
```diff
@@ -17,8 +17,11 @@ channel-type.system.location.description = Ort in geographischen Koordinaten (Br
 channel-type.system.motion.label = Bewegung
 channel-type.system.motion.description = Zeigt eine erkannte Bewegung an.
 channel-type.system.brightness.label = Helligkeit
+channel-type.system.brightness.description = Kontrollieren Sie die Helligkeit und schalten Sie das Licht ein und aus.
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties]: I would rather not address the user, i.e. "sie" directly, but instead describe it like on the other channels: Just what can be done with the channel. So here I would recommend: "Steuert die Helligkeit und schaltet das Licht ein und aus."

File: bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties [hunk 2]
```diff
@@ -17,8 +17,11 @@ channel-type.system.location.description = Ort in geographischen Koordinaten (Br
...
 channel-type.system.color.label = Farbe
+channel-type.system.color.description = Kontrolliere die Farbe des Lichts.
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties]: see above: "Steuert die Lichtfarbe."

File: bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties [hunk 3]
```diff
@@ -17,8 +17,11 @@ channel-type.system.location.description = Ort in geographischen Koordinaten (Br
...
 channel-type.system.color-temperature.label = Farbtemperatur
+channel-type.system.color-temperature.description = Kontrolliere die Farbtemperatur des Lichts.
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties]: "Steuert die Farbtemperatur des Lichts."

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java [hunk 4]
```diff
@@ -160,23 +160,26 @@
      */
     public static final ChannelType SYSTEM_BRIGHTNESS = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "brightness"), "Brightness", "Dimmer")
+            .withDescription("Control the brightness and switch the light on and off").withCategory("DimmableLight")
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java]: "Controls the brightness and switches the light on and off."

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java [hunk 5]
```diff
@@ -160,23 +160,26 @@
...
-            .withCategory("Light").build();
+            .build();
 
     /**
      * Color: default system wide {@link ChannelType} which allows changing the color
      */
     public static final ChannelType SYSTEM_COLOR = ChannelTypeBuilder
-            .state(new ChannelTypeUID(BINDING_ID, "color"), "Color", "Color").withCategory("ColorLight").build();
+            .state(new ChannelTypeUID(BINDING_ID, "color"), "Color", "Color")
+            .withDescription("Control the color of the light").withCategory("ColorLight").build();
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java]: "Control**s**...."

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java [hunk 6]
```diff
@@ -160,23 +160,26 @@
...
 
     /**
      * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature
      */
     public static final ChannelType SYSTEM_COLOR_TEMPERATURE = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "color-temperature"), "Color Temperature", "Dimmer")
+            .withDescription("Control the color temperature of the light").withCategory("ColorLight")
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java]: "Control**s**"...

File: bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/SystemWideChannelTypesTest.java [hunk 7]
```diff
@@ -224,30 +225,31 @@ public void systemChannelTypesShouldBeTranslatedProperly() {
...
-        assertNull(atmosphericHumidityChannelType.getDescription());
+        assertEquals("Aktuelle atmosphärische relative Luftfeuchtigkeit.",
+                atmosphericHumidityChannelType.getDescription());
 
         ChannelType barometricPressureChannelType = systemChannelTypeProvider
                 .getChannelType(BAROMETRIC_PRESSURE_CHANNEL_TYPE_UID, Locale.GERMAN);
         assertNotNull(barometricPressureChannelType);
         assertEquals("Luftdruck", barometricPressureChannelType.getLabel());
-        assertNull(barometricPressureChannelType.getDescription());
+        assertEquals("Aktueller luftdruck.", barometricPressureChannelType.getDescription());
```
Comment (NorbertHD) [inline bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/SystemWideChannelTypesTest.java]: There is a typo, "Luftdruck" starts with upper case.

**Comment by lolodomo** [comment]
Typo corrected and tests fixed.

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java [hunk 8]
```diff
@@ -160,23 +160,26 @@
      */
     public static final ChannelType SYSTEM_BRIGHTNESS = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "brightness"), "Brightness", "Dimmer")
+            .withDescription("Controls the brightness and switch the light on and off").withCategory("DimmableLight")
```
Comment (triller-telekom) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java]: switch**es**

**Comment by lolodomo** [comment]
Done.
Buid succeeded.

**Pull request merged**